samples/dapr/.tye/process_store

```diff
@@ -0,0 +1,6 @@
+{"pid":"80819"}
```

Comment:
Remove? We should add .tye to .gitignore

Reply:
yes. We should consider moving this to user profile if we have to as well.

Reply:
Sure the only thing I’m worried about is that it needs to be keyed by something so we can find it on the next run. If we don’t multiple runs by the same user can conflict

Reply:
A pretty simple way to do that is to key based on PID of the Host. So you get a hierarchy like:

```
$Home/.tye/pid/12345/<data we capture today>
$Home/.tye/pid/56789/<data we capture today>
```

Then the host can look for orphaned processed from another run by considering:

- pid of other running host process
- folders that exist
- (optional) timestamps of files

The only case where you'd have a problem is if the OS decided to reuse a PID for another tye host process.